src/server/common/model/answer/origin-type-other/origin-type-other.js

```diff
@@ -0,0 +1,36 @@
+import { escapeMarkdown } from '../../../helpers/escape-text.js'
+import { TextAnswer } from '../text/text.js'
+
+/** @import {TextConfig} from '../text/text.js' */
+
+/**
+ * @typedef {{ originTypeOther: string }} OriginTypeOtherPayload
+ */
+
+/**
+ * @augments {TextAnswer<OriginTypeOtherPayload>}
+ */
+export class OriginTypeOtherAnswer extends TextAnswer {
+  /** @type {TextConfig} */
+  static config = {
+    payloadKey: 'originTypeOther',
+    characterWidth: 20,
+    validation: {
+      empty: {
+        message: 'Enter the premises type'
+      },
+      maxLength: {
+        value: 5000,
+        message: 'Your answer must be no longer than 5000 characters'
+      }
+    }
+  }
+
+  get emailHtml() {
+    const data = this._data?.[this.config.payloadKey]
+    if (data.length === 0) {
```

Comment:
We can't have a zero-length case co-exist with a provided `validation.empty.message` in the config - so we can remove this. 

we should just be passing through the escapedMarkdown of the text answer